packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -40,11 +40,13 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
     double? maxWidth,
     double? maxHeight,
     int? imageQuality,
+    int? limit,
```

Comment:
Please, do not add functionality to this method, if you can avoid it. Everything that returns `PickedFile` has been removed from public access of the plugin since v1.0.0 (see [changelog](https://pub.dev/packages/image_picker/changelog#100))

Reply:
Removed